test/common/index.js

```diff
@@ -920,6 +925,7 @@ const common = {
   pwdCommand,
   requireNoPackageJSONAbove,
   runWithInvalidFD,
+  setHappyEyeballsEnabled,
```

Comment:
Documentation for this should be added to `test/README.md`.

Reply:
👍

Reply:
Whoops! I meant to say `test/common/README.md`!